components/form/FAutocompleteInput.vue

```diff
@@ -301,6 +308,25 @@ function formatOption(option: FMenuOption) {
   );
 }
 
+function handleMenuToggle(toggleValue: boolean) {
+  if (toggleValue) {
+    // When opening the menu
+    matchInputWithSelectedOption();
+  } else {
+    // When closing the menu
+    setTimeout(matchInputWithSelectedOption, 15); // Wait for Popper to properly close the menu before changing input value, to avoid seeing menu content breaks due to menu items changing
+  }
+}
+
+function matchInputWithSelectedOption() {
```

Comment:
Technically, you _match_ nothing, you just _set_ the input based on the current matched option
```suggestion
function setInputWithSelectedOption() {
```

Reply:
"setInput**With**selectedOption" seems strange to me though.. 🤔 

How about `formatInputValue` ? It's simpler and descriptive of its actual purpose